models/uniswap/ethereum/uniswap_v3_ethereum_proposals.sql

```diff
@@ -14,7 +15,7 @@
 {% set blockchain = 'ethereum' %}
 {% set project = 'uniswap' %}
 {% set project_version = 'v3' %}
-{% set dao_name = 'DAO: Uniswap' %}
+{% set dao_name = 'DAO_Uniswap' %}
```

Comment:
any reason this has to change it's format?

Reply:
no, it was just me getting confused if I should use quotes around vars or not, messing around with it and forgot to put it back to original state - reverted now